From b12504fad68d8cee8920f602762b5964b1a9f77f Mon Sep 17 00:00:00 2001
From: Zhou weixin <zwx@rock-chips.com>
Date: Thu, 25 Jan 2018 09:17:13 +0800
Subject: [PATCH] ARM: dts: rockchip: rk3126-bnd-d708: update system/vendor
 partition path

The new partition path is compatible with the following two partition names.
GTP partition: rknand0px
Old partition: rknand_X(rknand_system)

Change-Id: I993d42064bf4e093bfe01512db563f635baeff39
Signed-off-by: Weixin Zhou <zwx@rock-chips.com>
---
 arch/arm/boot/dts/rk3126-bnd-d708.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rk3126-bnd-d708.dts b/arch/arm/boot/dts/rk3126-bnd-d708.dts
index 3440b00af3e4..8e1340d88248 100644
--- a/arch/arm/boot/dts/rk3126-bnd-d708.dts
+++ b/arch/arm/boot/dts/rk3126-bnd-d708.dts
@@ -119,14 +119,14 @@
 				compatible = "android,fstab";
 				system {
 					compatible = "android,system";
-					dev = "/dev/block/rknand_system";
+					dev = "/dev/block/by-name/system";
 					type = "ext4";
 					mnt_flags = "ro,barrier=1,inode_readahead_blks=8";
 					fsmgr_flags = "wait";
 				};
 				vendor {
 					compatible = "android,vendor";
-					dev = "/dev/block/rknand_vendor";
+					dev = "/dev/block/by-name/vendor";
 					type = "ext4";
 					mnt_flags = "ro,barrier=1,inode_readahead_blks=8";
 					fsmgr_flags = "wait";
-- 
2.35.3

